pyfar/dsp/dsp.py

```diff
@@ -780,6 +783,12 @@ def regularized_spectrum_inversion(
             numerical aspects of linear inversion. Philadelphia: SIAM, 1998.
 
     """
+    # Deprecation warning for freq_range parameter
+    warnings.warn((
```

Comment:
```suggestion
if freq_range is not None:
    warnings.warn((
```
in this way the warning is just thrown if sth uses freq_range